src/Tests/Tests.csproj

```diff
@@ -10,13 +10,14 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Remove="NServiceBus.Newtonsoft.Json.approved.cs" />
-    <PackageReference Include="ApiApprover" Version="6.1.0-beta2" />
-    <PackageReference Include="ObjectApproval" Version="1.6.0" />
-    <PackageReference Include="PublicApiGenerator" Version="6.1.0-beta2" />
-    <PackageReference Include="NUnit" Version="3.7.1" />
-    <PackageReference Include="ApprovalTests" Version="3.0.13" />
-    <PackageReference Include="ApprovalUtilities" Version="3.0.13" />
-    <PackageReference Include="Mono.Cecil" Version="0.10.0-beta6" />
+    <Compile Remove="NServiceBus.Newtonsoft.Json.received.cs" />
```

Comment:
why are the approved/received files here ending with cs and not txt like the other packages?